src/dev/build/tasks/patch_native_modules_task.ts

```diff
@@ -40,19 +40,19 @@ interface Package {
 
 const packages: Package[] = [
   {
+    // Tip: use `scripts/download_re2.sh` to download binary artifacts from GitHub
```

Comment:
I think we should add the folder where this is being downloaded into the .gitignore

Reply:
I could also just download into `/tmp`?

Reply:
Just pushed a new commit that changed the download dir to `/tmp/re2`